src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Printeble, AreaCalculateble {
```

Comment:
if you implement Printable and AreaCalculateble into your Figure class you don't have to implement it in every separate figure class because they will inherit it